libs/novui/src/tokens/semanticColors.tokens.ts

```diff
@@ -16,20 +16,6 @@ export const LEGACY_COLOR_SEMANTIC_TOKENS = defineSemanticTokens.colors({
       type: 'color',
     },
   },
-  table: {
```

Comment:
This was duplicate see line 76

Reply:
Change was needed because of failing of here: https://github.com/novuhq/novu/actions/runs/9495865804/job/26169025237#step:6:53